package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sap-devx/webview-rpc",
-  "version": "0.2.3",
+  "version": "0.2.4",
```

Comment:
```suggestion
  "version": "0.3.0",
```

Reply:
major version change should be considered on breaking/incompatible change, not minor feature addition:
https://semver.org/

Reply:
exactly. in your case the patch number increased rather than minor.

Reply:
done